### PR TITLE
[READY] Localize last email

### LINKED ISSUE #273: @defactolearning.de email
het duitsland team will graag @defactolearning.de emails die dan ook in outlook terecht komen :)

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- localize email on `contact_info` too
- remove redundant phone numbers from locales
- use `unless` instead of `if`

Fixes #273

### CODE REVIEW

File: source/localizable/_footer.erb [hunk 1]
```diff
@@ -44,12 +44,10 @@
       <p>
         Zuiderpark 3<br>
         9724 AD Groningen<br>
-        <% if I18n.locale == :de %>
-          Niederlande
-        <% end %>
+        <%= "Niederlande" unless I18n.locale == :nl %>
```
Comment (fatboypunk) [inline source/localizable/_footer.erb]: I think an if would be better here, if the pages are going to be translated to english you'd have to change this as well

Comment (fatboypunk) [inline source/localizable/_footer.erb]: Nice that you made it a oneliner though :D

Comment (RonaldDijkstra) [inline source/localizable/_footer.erb]: @sn3p said a translated string would be best (empty in nl.yml).

Comment (fatboypunk) [inline source/localizable/_footer.erb]: he's right :)

Comment (RonaldDijkstra) [inline source/localizable/_footer.erb]: Ok

**Pull request merged**